src/Resources/app/storefront/src/storefront/js/nbbreturnforms.js

```diff
@@ -0,0 +1,39 @@
+const manufacturerSupportFormElementsNumber = 3;
+const productDetailsFormElementsNumber = 5;
+const manufacturerSupportForm = 'manufacturer_support_form';
+const productDetailsForm = 'product_details_form';
+const others = 'others';
```

Comment:
Maybe call this variable like "otherElementsKey" or something like that? So when comparising, you will be sure, what you're comparing to.

Reply:
My intention was use the same name as label in the template, as the value passed from select form. Going further by your suggestion I would like to change name to `otherProducts`.